src/main/java/com/epam/spring/homework2/bean/BaseBean.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.spring.homework2.bean;
+
+import com.epam.spring.homework2.validation.annotation.IsInRange;
+import com.epam.spring.homework2.validation.annotation.IsNotNull;
+
+public abstract class BaseBean {
```

Comment:
Does not contain abstract methods, thus should not be abstract. Also most likely can be package-private